.github/workflows/purge-azure-test-resources.yaml

```diff
@@ -108,7 +93,7 @@ jobs:
 
   create_issue_on_failure:
     name: Create issue for failing purge test resources run
-    needs: [purge_ghcr_dev, purge_azure_resources]
+    needs: [purge_azure_resources]
```

Comment:
Are we choosing to not create issue when purging containers fails since it seems to be just flaky?

Reply:
Nope, I just separated these two steps. They do different things.